src/mdbExtensionController.ts

```diff
@@ -314,6 +314,12 @@ export default class MDBExtensionController implements vscode.Disposable {
         );
       }
     );
+    this.registerParticipantCommand(
+      EXTENSION_COMMANDS.SHOW_EXPORT_TO_LANGUAGE_RESULTS,
+      (data: { content: string | null; language: string }) => {
+        return this._playgroundController.showExportToLanguageResults(data);
```

Comment:
Showing code from other languages in the playground files makes the syntax lack the correct highlighting. Have we thought about showing these another way? Like a preview pane in the language that is being exported to.

Reply:
Ah they did work, I think this was from the previous issue with cancelling.